src/theme/defaultTheme/index.ts

```diff
@@ -22,6 +24,7 @@ const theme = {
   iconButton,
   spinner,
   tag,
+  switch: Switch,
```

Comment:
use lowercase switch

Reply:
lowercase switch is reserved keyword, may be `_switch` is okay.

Reply:
Ohhhhhhhhhhh yeah yeah.